airflow/www/utils.py

```diff
@@ -338,7 +338,7 @@ def json_render(obj, lexer):
     return out
 
 
-def wrapped_markdown(s, css_class=None):
+def wrapped_markdown(s, css_class='rich_doc'):
```

Comment:
It was always `rich_doc` in 1.10.x , example:

https://github.com/apache/airflow/blob/1.10.15/airflow/www_rbac/utils.py#L316-L322

However, we pass `None` at few places, so just making it the default